doc/configuration.md

```diff
@@ -36,6 +36,7 @@ The Ignition configuration is a JSON document conforming to the following specif
     * **_mount_** (object): contains the set of mount and formatting options for the filesystem. A non-null entry indicates that the filesystem should be mounted before it is used by Ignition.
       * **device** (string): the absolute path to the device. Devices are typically referenced by the `/dev/disk/by-*` symlinks.
       * **format** (string): the filesystem format (ext4, btrfs, or xfs).
+      * **_useIfExists_** (boolean): whether or not the create operation shall be skipped if the target filesystem is already formatted with the desired filesystem type.
```

Comment:
Can be a minor bump, deprecate force and add the new option, mixing the two could be an error if you like. In the next major remove force.